ragna/assistants/_cohere.py

```diff
@@ -0,0 +1,92 @@
+import json
+from typing import AsyncIterator, cast
+
+from ragna.core import RagnaException, Source
+
+from ._api import ApiAssistant
+
+
+class CohereApiAssistant(ApiAssistant):
+    _API_KEY_ENV_VAR = "COHERE_API_KEY"
+    _MODEL: str
+    _CONTEXT_SIZE: int = 4_000
+    # See https://docs.cohere.com/docs/models#command
+
+    @classmethod
+    def display_name(cls) -> str:
+        return f"Cohere/{cls._MODEL}"
+
+    @property
+    def max_input_size(self) -> int:
+        return self._CONTEXT_SIZE
+
+    def _make_system_content(self) -> str:
```

Comment:
Let's call this 

```suggestion
    def _make_preamble(self) -> str:
```

to align our terminology here with Cohere. We can't really align across assistants built into Ragna given that every provider uses something different.

Reply:
Done